### ISSUE TITLE
flow-jump-to-definition: no such command 'edit ...'

### ISSUE THREAD

**Issue by jcowgar** [Delapouite/kakoune-flow#7]
When I place the cursor over a variable and execute the command flow-jump-to-definition, an error occurs and states:

1:1: 'flow-jump-to-definition' no such command: 'edit /correct/file/path/filename.js 4 7'

**Comment by Delapouite** [comment]
Right. I need to update this script to the latest breaking changes in kakoune.
